parmesan/distributions.py

```diff
@@ -1,13 +1,13 @@
 import math
 import theano.tensor as T
+c = - 0.5 * math.log(2*math.pi)
 
 
-c = - 0.5 * math.log(2*math.pi)
 def log_normal(x, mean, sd):
     """
     Compute log pdf of a Gaussian distribution with diagonal covariance, at values x.
 
-        .. math:: \log p(x) = \log \mathcal{N}(x; \mu, \sigma^2I)
+        .. math:: \log p(x) = \log \mathcal{N}(x; \mean, \sigma^2I)
```

Comment:
My  mistake. I just refactored. I'll fix that. 

I'm not sure what the best naming is. I think we should match numpy but i don't really like `sigma` 
from http://docs.scipy.org/doc/numpy/reference/generated/numpy.random.lognormal.html#numpy.random.lognormal